styles/globals.css

```diff
@@ -121,3 +121,29 @@ main li {
 main li span {
   flex: 0 0 7rem; 
 }
+
+.favorite-pin {
+  will-change: opacity, width;
+  opacity: 0;
+  width: 0;
```

Comment:
Hey, I am sorry for not making the requirement cleaner, but the desired outcome should be like:
<img width="449" alt="image" src="https://user-images.githubusercontent.com/16056918/199203007-a2fcacd0-c7a2-42d9-9405-061fbad354b5.png">


The pin is positioned absolutely to the heading. 
```
position: absolute;
left: -1.5rem;
```

Heading `h3` should be `position: relative` to make it work.

The pin is fully visible only when the feed was pinned.
By default, the pin is hidden, the same as the remove button.
On heading hover, the pin has an opacity of 0.5, but it stays in the same position:

```
position: absolute;
left: -1.5rem;
opacity: 0.5;
```

Hope you understand the requirement. 
Let me know if you still want to finish this (or I can do the changes on top of your PR)!
😊

Reply:
Sure 😄 
I'll finish it in a bit